vm/src/stdlib/json.rs

```diff
@@ -213,11 +209,14 @@ fn loads(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     // TODO: Implement non-trivial deserialisation case
     arg_check!(vm, args, required = [(string, Some(vm.ctx.str_type()))]);
     // TODO: Raise an exception for deserialisation errors
-    let kind: PyObjectKind = match string.borrow().kind {
-        PyObjectKind::String { ref value } => serde_json::from_str(&value).unwrap(),
+    let de = PyObjectDeserializer { ctx: &vm.ctx };
+    // TODO: Support deserializing string sub-classes
```

Comment:
We can use here isinstance and objstr::get_value right?

Reply:
So the arg_check! call means we know we have a string sub-class; I've modified us to use `objstr::get_value` as suggested, but we still aren't actually handling string sub-classes; #130 means that we _appear_ to be (per the test I have added), but that's because string sub-classing is currently broken.